File: my-leetcode-solutions/valid-parentheses.ts
// My solution to 20 - Valid Parentheses

function isValid(s: string): boolean {
    
};

// Time Complexity:

const testCases = [
    "()",
    "()[]{}",
    "(]",
    "([])",
    "([)]"
];

for (let i = 0; i < testCases.length; i++) {
    const output = isValid(testCases[i]);
    console.log(`Output: ${output} for input: ${testCases[i]}`);
}

export {};

File: my-leetcode-solutions-py/valid-parentheses.py
# My solution to 20 - Valid Parentheses

class Solution:
    def isValid(self, s: str) -> bool:
        pass


# Time Complexity:

s_test_cases = ["()", "()[]{}", "(]", "([])", "([)]"]

solution = Solution()

for i in range(len(s_test_cases)):
    output = solution.isValid(s_test_cases[i])
    print(f"Output: {output!r} for s as {s_test_cases[i]!r}")
